### ISSUE TITLE
Cannot change resolution of default fbo

### ISSUE THREAD

**Issue by stacksmith** [cbaggers/cepl#121]
I now spent way too much time on this... Even though the docs say "The default viewport can be modified, so feel free to update the resolution if the window size changes" I can't seem to do it.  I tried setf'ing it every way imaginable.  I just get nil from REPL, and (current-viewport) is still the same.  Placing setf's in various locations inside the render loop does not seem to affect it either.

It also appears that (eq (current-viewport) (current-viewport)) is nil.  Is current-viewport something that's concocted every render-frame, based on some sdl2 version of window size?  I noticed that there is a callback for window size deep in host, and am not sure how to get to it after a window is up...

Digging in, current-viewport seems to create a viewport from an FBO based on its size?  So do I need to resize the buffer?  How?

There seems to be an installable user routine to get the window size, but it is never called? 

In short, I am very confused...

**Comment by stacksmith** [comment]
I wrapped step-demo with   (with-viewport (make-viewport *window-size*) ...)
and set *window-size* whenever it changes...

Is that the right way to do that?

**Comment by cbaggers** [comment]
Not being able to modify the default fbo is definitely a bug. I'll look into this.

The code you showed will work but then you are creating a viewport object every frame. I'd store that somewhere. Saying that, you wouldnt be in this place if you could mutate the default fbo. So I'll get on that.

**Comment by stacksmith** [comment]
@cbaggers, please clarify the ` (eq (current-viewport) (current-viewport)) ` issue.  Is curent-viewport concocted on-demand, or am I missing something?

**Comment by cbaggers** [comment]
Have you tested this since my changes? Seems to work here.

```
CL-USER> (cepl:repl)
New context v4.1
-----------------
    CEPL-REPL    
-----------------

CL-USER> (in-package :cepl)
#<PACKAGE "CEPL">

CEPL> (eq (current-viewport) (current-viewport))
T
```
